mintpy/plot_network.py

```diff
@@ -76,6 +77,19 @@ def create_parser():
     coh.add_argument('-v', '--vlim', nargs=2, type=float, default=(0.2, 1.0),
                      help='display range')
 
+    # Display baseline in the network plot
+    coh = parser.add_argument_group('Display Baseline', 'Color lines in the network plot with baseline instead of coherence')
+    coh.add_argument('--tb', '--tbase', dest='tbColor', action='store_true',
+                     help='color using temporal baseline')
+    coh.add_argument('--pb', '--pbase', dest='pbColor', action='store_true',
+                     help='color using perpendicular baseline')
+    coh.add_argument('--tbv', '--tbvlim', dest='tbvlim', nargs=2, type=float, default=(0, 365.25),
+                     help='display color range for temporal baseline [day]. (default: %(default)s)')
+    coh.add_argument('--pbv', '--pbvlim', dest='pbvlim', nargs=2, type=float, default=(0, 180.0),
+                     help='display color range for perpendicular baseline [meter]. (default: %(default)s)')
+    coh.add_argument('-c2', '--colormap2', dest='cmap_name2', default='RdYlBu_r',
```

Comment:
It seems like a new `--colormap2` option is not necessary, we could use/share the same `-c` for this purpose. Do you agree?

Reply:
Ok, this and the following related lines have been removed.